server/src/test/java/io/crate/expression/scalar/ExtractFunctionsTest.java

```diff
@@ -65,54 +77,72 @@ public void testExtractCenturyFromTimestampWithoutTimeZone() {
     @Test
     public void testExtractYear() throws Exception {
         assertEvaluate("extract(year from timestamp_tz)", 2014);
+        assertEvaluate("extract(year from INTERVAL '1250 days 49 hours')", 0);
+        assertEvaluate("extract(year from INTERVAL '14 years 1250 days 49 hours')", 14);
     }
 
     @Test
     public void testExtractQuarter() throws Exception {
         assertEvaluate("extract(quarter from timestamp_tz)", 1);
+        assertEvaluate("extract(month from INTERVAL '14 years 58 months 1250 days 49 hours')", 10);
+        assertEvaluate("extract(month from INTERVAL '58 months 1250 days 49 hours')", 10);
```

Comment:
shoule they be `quarter`?

Reply:
yes, thx! leftover while testing various combinations.